src/components/cc-env-var-form/cc-env-var-form.js

```diff
@@ -40,17 +40,11 @@ export class CcEnvVarForm extends LitElement {
       addonName: { type: String, attribute: 'addon-name' },
       appName: { type: String, attribute: 'app-name' },
       context: { type: String, reflect: true },
-      error: { type: Boolean, reflect: true },
       heading: { type: String, reflect: true },
-      parserOptions: { type: Object, attribute: 'parser-options' },
       readonly: { type: Boolean, reflect: true },
       restartApp: { type: Boolean, attribute: 'restart-app' },
-      saving: { type: Boolean, reflect: true },
-      variables: { type: Array },
+      variables: { type: Object },
       _currentVariables: { type: Array, state: true },
-      _description: { type: String, state: true },
-      _expertVariables: { type: Array, state: true },
-      _jsonVariables: { type: Array, state: true },
```

Comment:
**🔨 issue:** I think that removing those two dedicated variables broke the expert and JSON modes. I can't update values in [the stories](https://clever-components-preview.cellar-c2.services.clever-cloud.com/new-smart-design/cc-env-var-form/index.html?path=/story/%F0%9F%9B%A0-environment-variables-cc-env-var-form--default-story).